server/protocols/webadmin/webadmin-data/src/test/java/org/apache/james/webadmin/routes/UserMappingRoutesTest.java

```diff
@@ -150,18 +127,18 @@ void getUserMappingsShouldReturnCorrespondingMappingsFromUsername() throws Excep
     @Test
     void getUserMappingsShouldReturnSameMappingsWhenParametersInUpperCase() throws RecipientRewriteTableException {
         recipientRewriteTable.addAddressMapping(
-            MappingSource.fromUser(User.fromUsername("alice123@domain.tld")), "user123@domain.tld");
+            MappingSource.fromUser(User.fromUsername(ALICE_ADDRESS)), "user123@domain.tld");
         recipientRewriteTable.addAliasMapping(
-            MappingSource.fromUser(User.fromUsername("alice123@domain.tld")), "aliasuser123@domain.tld");
+            MappingSource.fromUser(User.fromUsername(ALICE_ADDRESS)), "aliasuser123@domain.tld");
         recipientRewriteTable.addGroupMapping(
-            MappingSource.fromUser(User.fromUsername("alice123@domain.tld")), "group123@domain.tld");
+            MappingSource.fromUser(User.fromUsername(ALICE_ADDRESS)), "group123@domain.tld");
 
         recipientRewriteTable.addAddressMapping(
-            MappingSource.fromUser(User.fromUsername("bob456@domain.tld")), "user456@domain.tld");
+            MappingSource.fromUser(User.fromUsername(BOB_ADDRESS)), "user456@domain.tld");
         recipientRewriteTable.addAliasMapping(
-            MappingSource.fromUser(User.fromUsername("bob456@domain.tld")), "aliasuser456@domain.tld");
+            MappingSource.fromUser(User.fromUsername(BOB_ADDRESS)), "aliasuser456@domain.tld");
         recipientRewriteTable.addGroupMapping(
-            MappingSource.fromUser(User.fromUsername("bob456@domain.tld")), "group456@domain.tld");
+            MappingSource.fromUser(User.fromUsername(BOB_ADDRESS)), "group456@domain.tld");
```

Comment:
you can as well extract your user, aliasuser and group emails into constants, as you use them in two tests